app/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     // Retrofit
     implementation "com.squareup.retrofit2:retrofit:$rootProject.retrofit_version"
     implementation "com.squareup.retrofit2:converter-gson:$rootProject.retrofit_version"
+    implementation "com.squareup.moshi:moshi:1.11.0"
```

Comment:
Please use the usual dependency version specification style that has been used for others